python/metrics/tests/test_events.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import numpy as np
+import hydrotools.metrics.events as emet
+
+one_series = np.ones(10)
+range_series = np.arange(9)
+negative_series = -one_series
+empty_series = np.empty(0)
```

Comment:
1. Add type hints
2. Add triple quote docstrings underneath each describing what they represent and what they are used for
3. capitalize since they are global constants